syntax = "proto3";

message DeviceInfo {
  string bootloader = 1;
  string proc_version = 2;
  string codename = 3;
  string incremental = 4;
  string fingerprint = 5;
  string boot_id = 6;
  string android_id = 7;
  string base_band = 8;
  string inner_version = 9;
}
